packages/common/src/core/_internal/PropertyMapper.ts

```diff
@@ -104,6 +118,11 @@ export function mapModelToObject<T>(
   object: T,
   mappings: IPropertyMap[]
 ): T {
+  if (model.item.typeKeywords?.includes(CANNOT_DISCUSS)) {
```

Comment:
Maybe we should move [isDiscussable](https://github.com/Esri/hub.js/blob/master/packages/discussions/src/utils/posts/index.ts#L61) utility into `hub-common` similar to how we moved the `CANNOT_DISCUSS` constant and use that here. We probably want to avoid duplicating that logic in a lot of places in case we pivot to a different strategy of determining if something is discussable or not.

```jsx
object.isDiscussable = isDiscussable(model.item);
```